drivers/gles3/storage/texture_storage.cpp

```diff
@@ -1197,6 +1209,14 @@ void TextureStorage::_update_render_target(RenderTarget *rt) {
 		return;
 	}
 
+	Config *config = Config::get_singleton();
+#if !defined(GLES_OVER_GL) && !defined(WEB_ENABLED)
+	PFNGLFRAMEBUFFERTEXTUREMULTIVIEWOVRPROC glFramebufferTextureMultiviewOVR = nullptr;
+	if (config->multiview_supported && rt->view_count > 1) {
+		glFramebufferTextureMultiviewOVR = (PFNGLFRAMEBUFFERTEXTUREMULTIVIEWOVRPROC)eglGetProcAddress("glFramebufferTextureMultiviewOVR");
+	}
+#endif
+
```

Comment:
Same comment as above. Initialize this once and then share between places that need it.